stdlib/3/hmac.pyi

```diff
@@ -0,0 +1,27 @@
+# Stubs for hmac (Python 3.5)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from typing import Any
+from _operator import _compare_digest as compare_digest
```

Comment:
Hmm, I guess I should be running typeshed's own tests then (the python 2 tests fail for me locally but I haven't touched anything there. It's all sqlalchemy related). I wonder why my own usage of this module only required the stub to exist and not to work. 

I've added a stub for `_operator`. I couldn't do the same for `operator` because stubgen hits a parse error in `operator.py` (the matrix multiplication operator).